configs/opencv-ovms/gst_capi/pipelines/capi_yolov5/main.cpp

```diff
@@ -0,0 +1,1144 @@
+	}
+}
+
+// TODO: Multiple references state that imshow can't be used in any other thread than main!
+void displayGUIInferenceResults(cv::Mat analytics_frame, std::vector<DetectedResult> &results, int latency, int througput)
+{
+    auto ttid = std::this_thread::get_id();
+    std::stringstream ss;
+    ss << ttid;
+    std::string tid = ss.str();
+
+    const float scaler_w = _window_width/416.0f;
+    const float scaler_h = _window_height/416.0f;
+
+    for (auto & obj : results) {
+	    const float x0 = obj.x;
+        const float y0 = obj.y;
+        const float x1 = obj.x + obj.width;
+        const float y1 = obj.y + obj.height;
+        const float xc = (x1-x0)/2.0f + x0;
+        const float yc = (y1-y0)/2.0f + y0;
+
+        const float scaled_x0 = xc-obj.width*scaler_w/2.0f;
+        const float scaled_y0 = yc-obj.height*scaler_h/2.0f;
+
+        const float scaled_x1 = xc+obj.width*scaler_w/2.0f;
+        const float scaled_y1 = yc+obj.height*scaler_h/2.0f;
+
+        // printf("x0, y0: [%f] [%f] x1, y1: [%f] [%f]", x0, y0, x1, y1);
+
+        cv::rectangle( analytics_frame,
+            cv::Point( (int)(scaled_x0),(int)(scaled_y0) ),
+            cv::Point( (int)(scaled_x1), (int)(scaled_y1)),
+            cv::Scalar(255, 0, 0),
+            2, cv::LINE_8 );
+    } // end for
+
+    //latency
+    // std::string fps_msg = (througput == 0) ? "..." : std::to_string(througput) + "fps";
+    // std::string latency_msg = (latency == 0) ? "..." :  std::to_string(latency) + "ms";
+    // std::string roiCount_msg = std::to_string(results.size());
+    // std::string message = "E2E Pipeline Performance: " + latency_msg + " and " + fps_msg + " with ROIs#" + roiCount_msg;
+    // cv::putText(analytics_frame, message.c_str(), cv::Size(0, 20), cv::FONT_HERSHEY_PLAIN, 1, (255, 0, 0), 1, cv::LINE_4);
+    // cv::putText(analytics_frame, tid, cv::Size(0, 40), cv::FONT_HERSHEY_PLAIN, 1, (255, 0, 0), 1, cv::LINE_4);
+
+    cv::Mat presenter;
+
+    {
+        std::lock_guard<std::mutex> lock(_drawingMtx);
+        cv::imshow("OpenVINO Results " + tid, analytics_frame);
+        cv::waitKey(1);
+    }
+}
+
+void saveInferenceResultsAsVideo(cv::Mat &presenter, std::vector<DetectedResult> &results)
+{
+    for (auto & obj : results) {
+        const float scaler_w = 416.0f/_window_width;
+        const float scaler_h = 416.0f/_window_height;
+        //std::cout << " Scalers " << scaler_w << " " << scaler_h << std::endl;
+        //std::cout << "xDrawing at " << (int)obj.x*scaler_w << "," << (int)obj.y*scaler_h << " " << (int)(obj.x+obj.width)*scaler_w << " " << (int) (obj.y+obj.height)* scaler_h << std::endl;
+    
+        cv::rectangle( presenter,
+         cv::Point( (int)(obj.x*scaler_w),(int)(obj.y*scaler_h) ),
+         cv::Point( (int)((obj.x+obj.width) * scaler_w), (int)((obj.y+obj.height)*scaler_h) ),
+         cv::Scalar(255, 0, 0),
+         4, cv::LINE_8 );  
+    } // end for
+    cv::imwrite("result.jpg", presenter);
+}
```

Comment:
is this one used? the scaler_w and scaler_h seems backwards.  if not used please remove this function